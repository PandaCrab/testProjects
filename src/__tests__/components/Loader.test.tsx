import { mount } from 'enzyme';

import Loader from '../../components/Loader';

it('should render loader', () => {
    expect(mount(<Loader />)).toMatchSnapshot();
})
