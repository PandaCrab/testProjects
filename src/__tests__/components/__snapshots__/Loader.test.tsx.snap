// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render loader 1`] = `
<Loader>
  <Spinner
    animation="border"
    role="status"
  >
    <div
      className="spinner-border"
      role="status"
    >
      <span
        className="visually-hidden"
      >
        Loading...
      </span>
    </div>
  </Spinner>
</Loader>
`;
